praetorian_cli/ui/aegis/commands/schedule_helpers.py

```
@@ -0,0 +1,296 @@
+        return None, None
+    except Exception:
+        return None, None
+
+
+def interactive_schedule_picker(menu, suggested=None, prompt_prefix="schedule"):
+    """Interactive schedule picker with fuzzy search.
+
+    If suggested is provided and matches a schedule prefix, returns it directly.
+    Otherwise shows an inline fuzzy picker.
+    """
+    colors = getattr(menu, 'colors', DEFAULT_COLORS)
```

Ruff (0.14.14)

[error] Local variable `colors` is assigned to but never used

Remove assignment to unused variable `colors`

(F841)